packages/rich-text/src/SyncEditorChanges.tsx

```diff
@@ -25,7 +25,8 @@ const useAcceptIncomingChanges = (incomingValue?: Value) => {
     }
 
     lastIncomingValue.current = incomingValue;
-    setEditorValue(editor, incomingValue);
+    // FIXME see why this casting is needed
```

Comment:
something we should do before the merging?

Reply:
We add the tracking property to our editor, with the new way things are structured at the top level we don't get the same type here unfortunately. I move the casting to where the editor is declared